R/random-forest-development.R

```diff
@@ -26,6 +25,41 @@
 #' F leads to removal of rows containing NULLs.
 #' @param debug Provides the user extended output to the console, in order
 #' to monitor the calculations throughout. Use T or F.
+#' @param cores Number of cores you'd like to use. Defaults to 2.
+#' @param trees Number of trees in the forest. Defaults to 201.
```

Comment:
"trees" is mentioned as a default param but then not demonstrated in example. might be good to include in example. also, the option to adjust the trees param should probably be available in deploy if it's available in develop.